build.xml

```diff
@@ -212,7 +219,6 @@
                mainclassname="${APP_JAVA_CLASS}">
 
       <arch name="x86_64" />
-      <arch name="i386" />
```

Comment:
I imagine "i386" is mostly obsolete at this point?

Reply:
We're not trying to support a universal binary -- there are too many differences between the 32/64-bit builds, including JDK version -- so 64-bit support is all we need.

Reply:
Got it.  So all 32 bit references are presumably obsolete for the Mac build as 32 bit builds are no longer supported.

Reply:
Right, of course.